lib/vsc/mympirun/rm/sched.py

```diff
@@ -49,6 +49,14 @@ def what_sched(requested):
                 return sched, found_sched
         LOGGER.warn("%s scheduler was requested, but mympirun failed to find an implementation", requested)
 
+    # if PBS_NODEFILE is not present, use local scheduler
+    filename = os.environ.get('PBS_NODEFILE', None)
+    if filename is None:
```

Comment:
only used in one place, so no need for a variable

also, `os.environ.get` uses `None` as fallback value by default, no need to specify it

and usually `os.getenv` is used instead (same behaviour)

```python
if not os.getenv('PBS_NODEFILE'):
```